src/organisms/tables/ComparisonTable/comparison_table.module.scss

```diff
@@ -0,0 +1,63 @@
+@import 'src/assets/stylesheets/global';
+
+.comparison-table {
+  border: 1px solid $neutral-5;
+  border-radius: 6px;
+  padding: 0;
+
+  .child-wrapper:last-child {
+    .row {
+      border-bottom: none;
+    }
+  }
+}
+
+.header {
+  padding: 24px 30px;
+  border-right: 1px solid $neutral-5;
+  border-bottom: 1px solid $neutral-5;
+
+  &:last-child {
+    border-right: none;
+  }
+}
+
+.row {
+  padding: 18px 30px;
+  position: relative;
+  border-right: 1px solid $neutral-5;
+  border-bottom: 1px solid $neutral-5;
+
+  &:last-child {
+    border-right: none;
+  }
+}
+
+.layout {
+  padding: 0;
+}
+
+@media #{$small-only} {
+  .comparison-table {
+    border: none;
+  }
+
+  .header {
+    border: none;
+
+    &:not( :first-child ) {
+      display: none;
```

Comment:
@cisacke Missed this before--what is this for? We should try to avoid using this `:not(:pseduo` selector -- it's virtually the most specific selector you can write, so it's almost impossible to override it without using important.